src/pages/en/developers/apis/1-routing-api/itinerary-planning/index.md

```diff
@@ -291,109 +236,173 @@ Itinerary planning can be tuned by multiple arguments of the **plan** query.
 
 ```graphql
 {
-  plan(
-    from: {lat: 60.1713572, lon: 24.9416544}
-    to: {lat: 60.29280, lon: 25.04396}
-    numItineraries: 3
-    allowedTicketTypes: "HSL:AB"
+  planConnection(
+    origin: {location: {coordinate: {latitude: 60.199196699999995, longitude: 24.9397302}}}
+    destination: {location: {coordinate: {latitude: 60.168438, longitude: 24.929283}}}
+    first: 2
   ) {
-    date
-    itineraries {
-      legs {
-        startTime
-        endTime
-        mode
-        route {
-          gtfsId
-        }
-        from {
-          name
-          stop {
-            zoneId
+    pageInfo {
+      endCursor
+    }
+    edges {
+      node {
+        start
+        end
+        legs {
+          fareProducts {
+            product {
+              id
+              name
+            }
```

Comment:
please add into product the complex part:
                ... on DefaultFareProduct {                                                                                  
                  price {                                                                                                    
                    amount                                                                                                   
                  }